lenet/main.js

```diff
@@ -37,6 +37,11 @@ function getInputFromCanvas() {
   return input;
 }
 
+function getMedianValue(arr) {
+  return arr.length % 2 !== 0 ? arr[Math.floor(arr.length / 2)] :
+      (arr[arr.length / 2 - 1] + arr[arr.length / 2]) / 2;
```

Comment:
I suppose the array should be sorted before get the media value, am I correct?

Reply:
Thanks @huningxin 
It's fixed in new commit, please take another look, thanks.